packages/wine.rb

```diff
@@ -90,6 +96,7 @@ def self.postinstall
   end
 
   def self.remove
+    @xdg_config_home = ENV['XDG_CONFIG_HOME']
```

Comment:
What happens when the environment variable isn't set?  Should we have a default fallback value?

Reply:
How about `@xdg_config_home =  "#{CREW_PREFIX}/.config" if @xdg_config_home.to_s.empty?`

Reply:
That works.  I'd hate to recursively remove the wrong `.wine` folder. :)